### PR TITLE
Dev/9 typhoon

### PR DESCRIPTION
**Pull request by clpetersonucf**
- Widget URLs in the install config now default to `latest` instead of hard-coded, outdated versions.
- Fix for local test failure
- Access Request: a LTI-embedded widget that is accessed by an instructor who does not own a widget now provides a list of widget owners, and the option to request access from any of them. The selected widget owner is sent a notification to provide access to the requesting user
- The `phpcs` git pre-commit hook now defaults to a `full` report instead of `summary`
- browsersync removed, with prejudice
- Added server-side support for `obj` media file types
- Default ASSET_STORAGE_DRIVER in the `docker-compose.yml` is now `file` instead of `s3`
- Added API endpoint to verify instance access permissions
- A big ol' pile of dependabot updates

** NOTE: THIS PR INCLUDES A MIGRATION **

### CODE REVIEW

File: docker/docker-compose.override.test.yml [hunk 1]
```diff
@@ -31,8 +31,8 @@ services:
       - MYSQL_USER
       - MYSQL_PASSWORD
       - MYSQL_DATABASE=test
-    volumes:
-      - "./config/mysql/dev.my.cnf:/etc/mysql/conf.d/my.cnf"
+    # volumes:
+    #   - "./config/mysql/dev.my.cnf:/etc/mysql/conf.d/my.cnf"
```
Comment (FrenjaminBanklin) [inline docker/docker-compose.override.test.yml:35]: Might prefer just removing this entirely if they're no longer necessary.

File: fuel/app/classes/materia/widget/asset.php [hunk 2]
```diff
@@ -271,6 +273,13 @@ public function render(string $size): void
 			header("Content-Length: {$bytes}");
 			header('Content-Transfer-Encoding: binary');
 			header('Cache-Control: max-age=31536000');
+
+			// Special case for 3D labeling
```
Comment (FrenjaminBanklin) [inline fuel/app/classes/materia/widget/asset.php:277]: A little more context might be necessary here.

**Review by FrenjaminBanklin — APPROVED** [review]
I'm a little wary of always using the 'latest' version of a widget by default. I vaguely remember having this conversation years ago and settling on hard-coding the versions intentionally to avoid some pitfall or another. Maybe everything has stabilized now to the point that we don't need to worry about it.

There are a couple of spots where more clarity in comments would be nice, but otherwise if we're okay with the departure from installing specific widget versions, I see no reason to hold this up.

File: fuel/app/classes/materia/widget/asset.php (hunk 2 above)
Comment (clpetersonucf) [reply fuel/app/classes/materia/widget/asset.php]: Updated comment with additional context.

File: docker/docker-compose.override.test.yml (hunk 1 above)
Comment (clpetersonucf) [reply docker/docker-compose.override.test.yml]: Removed the lines in question.

**Review by ljoks — APPROVED** [review]
Just a bit of a lack of comments for new functions but overall lgtm

**Pull request merged**